src/main/java/uk/gov/mca/beacons/api/domain/Note.java

```diff
@@ -20,7 +20,7 @@
   private String text;
   private NoteType type;
   private LocalDateTime createdDate;
-  private UUID personId;
+  private UUID userAuthId;
```

Comment:
Could this be `personId`, `userId`, or `actorId` to be consistent with the `DeleteBeaconRequestDTO` which represents the user who created that note regardless of where they came from (B2C/Azure AD)